pkgs/development/tools/rover/default.nix

```diff
@@ -3,32 +3,47 @@
 , fetchFromGitHub
 , perl
 , rustPlatform
+, darwin
+, openssl
+, stdenv
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "rover";
-  version = "0.5.1";
+  version = "0.11.0";
 
   src = fetchFromGitHub {
     owner = "apollographql";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-wBHMND/xpm9o7pkWMUj9lEtEkzy3mX+E4Dt7qDn6auY=";
+    sha256 = "sha256-Ei6EeM0+b3EsMoRo38nHO79onT9Oq/cfbiCZhyDYQrc=";
   };
 
-  cargoSha256 = "sha256-n0R2MdAYGsOsYt4x1N1KdGvBZYTALyhSzCGW29bnFU4=";
+  cargoSha256 = "sha256-+iDU8LPb7P4MNQ8MB5ldbWq4wWRcnbgOmSZ93Z//5O0=";
+
+  buildInputs = [ openssl ] ++ lib.optionals stdenv.isDarwin [
```

Comment:
Why is openssl needed?

Reply:
shoot, it's not! that was a sloppy copy/paste 😬

Reply:
waiting for a build on another machine to doublecheck x86 builds and then I'll push a commit removing `openssl`